src/components/RoutingDiagram/__snapshots__/RoutingDiagram.test.tsx.snap

```diff
@@ -3,45 +3,45 @@
 exports[`renders multi route 1`] = `
```

Comment:
wait why did these change?

Reply:
I regenerated the snapshots because the html class names had just barely changed... should that not have happened?

Reply:
it shouldn't have. is this branch up to date against main?

Reply:
and are your local node_modules up to date?

Reply:
Oh didn't realize that'd affect them thank you, I've been working in another branch and switched back without yarn installing. I'll reinstall, checkout the original snaps and make sure they run